lib/dry/view/renderer.rb

```diff
@@ -31,6 +34,17 @@ def call(template, scope, &block)
         end
       end
 
+      def partial(template, scope, &block)
```

Comment:
Looks like `partial` and `call` are identical except for the `lookup` part, maybe we could DRY up the methods. What do you think?

Reply:
@GustavoCaso Yeah, I might try extracting something shared from these two methods when I wrap up this work. But I push too hard, since I don't mind a bit of repetition if it helps things remain clear (and we're only ever going to have these 2 methods here, not any more).